sections/announcement-bar.liquid

```diff
@@ -63,6 +63,26 @@
           "default": "accent-1",
           "label": "t:sections.all.colors.label"
         },
+        {
+          "type": "select",
+          "id": "text_alignment",
+          "options": [
+            {
+              "value": "left",
+              "label": "t:sections.announcement-bar.blocks.announcement.settings.text_alignment.options__1.label"
+            },
+            {
+              "value": "center",
+              "label": "t:sections.announcement-bar.blocks.announcement.settings.text_alignment.options__2.label"
+            },
+            {
+              "value": "right",
+              "label": "t:sections.announcement-bar.blocks.announcement.settings.text_alignment.options__3.label"
+            }
+          ],
+          "default": "center",
+          "label": "t:sections.announcement-bar.blocks.announcement.settings.text_alignment.label"
+        },
```

Comment:
Let's put it right under the `Text` setting to follow our latest content hierarchy ([Internal Figma file](https://www.figma.com/file/igRnZQceCieIPmM3NYZJyC/Flagship-Themes-Settings-UI?node-id=1093%3A16090))

Color scheme would also become the last to follow those logic too. Here is a reference
![](https://screenshot.click/07-49-eu3hg-bkn4i.png)

cc. @katycobb for visibility